kci_rootfs

```diff
@@ -59,6 +59,43 @@ class cmd_list_configs(Command):
         return True
 
 
+class cmd_list_variants(Command):
+    help = "Lists all rootfs variants"
+    opt_args = [Args.config, Args.arch]
+
+    def __call__(self, config_data, args, **kwargs):
```

Comment:
Can we come up with something more concise. Maybe something like:

```python
        if config_name and config_name not in rootfs_configs:
            print("{} : invalid config entry".format(config_name))
            return False
        if arch and config_name and arch not in rootfs_configs[config_name].arch_list:
            print("{} : invalid arch entry".format(arch))
            return False

        configs = (config for config in rootfs_configs if config == config_name or config_name is None)
        for config in configs:
            for arch_type in rootfs_configs[config].arch_list:
                if arch_type == arch or arch is None:
                    print(config, arch_type)
        return True
```

Reply:
ok, modified the command and implemented as stated above and tested.